uniffi_bindgen/src/bindings/gecko_js/templates/InterfaceTemplate.cpp

```diff
@@ -39,6 +46,16 @@ JSObject* {{ obj.name()|class_name_cpp(context) }}::WrapObject(
   return dom::{{ obj.name()|class_name_cpp(context) }}_Binding::Wrap(aCx, this, aGivenProto);
 }
 
+using nimbus_1725_MozPromise = MozPromise < {{context.ffi_rustbuffer_type()}}, {{context.ffi_rusterror_type()}}, false >;
```

Comment:
There is an `ffi_namespace()` method on the `ComponentInterface` struct that will produce this `nimbus_1725` prefix for you.

Reply:
Good to know; thanks!